cmake/configs/modes.cmake

```diff
@@ -130,6 +130,9 @@ else()
     endif()
 endif()
 
+# Address sanitizer builds should have debug info available and optimization off
```

Comment:
Nitpick: the comment applies to all sanitiser builds, not just ASAN

Reply:
Thanks - fixed.